strawberry/extensions/tracing/opentelemetry.py

```diff
@@ -101,6 +101,17 @@ def filter_resolver_args(
             return args
         return self._arg_filter(deepcopy(args), info)
 
+    def convert_to_allowed_types(self, value: Any) -> Any:
+        if isinstance(value, (bool, str, bytes, int, float)):
+            return value
+        elif isinstance(value, (list, tuple)):
+            return self.convert_list_or_tuple_to_allowed_types(value)
+        else:
+            return str(value)
+
+    def convert_list_or_tuple_to_allowed_types(self, value: list or tuple) -> str:
+        return ", ".join(map(str, map(self.convert_to_allowed_types, value)))
```

Comment:
Do we need a second map to str since by design the main function returns a string? Can we directly join after the first map?